base/changes.txt

```diff
@@ -6,6 +6,9 @@ to completeness or accuracy and it contains some references to files that are
 not part of the distribution.
 ================================================================================
 
+2024-09-03 Ulrike Fischer <Ulrike.Fischer@latex-project.org>
+	* fontdef.dtx: preload T1cmss.fd and T1cmtt.fd
```

Comment:
When referring to the .fd files, maybe use the letter case that matches the actual file names.
```suggestion
	* fontdef.dtx: preload t1cmss.fd and t1cmtt.fd
```